SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -58,14 +60,14 @@ void USpatialReceiver::OnCriticalSection(bool InCriticalSection)
 
 void USpatialReceiver::EnterCriticalSection()
 {
-	UE_LOG(LogTemp, Log, TEXT("SpatialReceiver: Entering critical section."));
+	UE_LOG(LogSpatialReceiver, Log, TEXT("SpatialReceiver: Entering critical section."));
```

Comment:
Remove `SpatialReceiver:` as this is covered by the category,

Reply:
+1